### PR TITLE
Doc: add depth parameter

### LINKED ISSUE #702: Depth option
Hi,

The depth option is a really important and useful feature but it's missing from the doc.
I wanted to get all stops from a specfic route and find out about this option by someone on IRC.

Exemple of the depth option :
https://api.navitia.io/v1/coverage/fr-idf/routes/route:RTP:1093630?depth=3

Can you please update the doc ?

### PR DESCRIPTION
**Pull request by stifoon**
close https://github.com/CanalTP/navitia/issues/702

### CODE REVIEW

**Comment by kinnou02** [comment]
jenkins, test this please

File: documentation/navitia.io/source/integration.rst [hunk 1]
```diff
@@ -362,6 +362,41 @@ Specific parameters
...
+* Get the line 1 id 
+
+	* https://api.navitia.io/v1/coverage/fr-idf/pt_objects?q=metro%201
+	* the id is "line:RTP:1197611"
+
+* Get routes for this line 
+
+	* https://api.navitia.io/v1/coverage/fr-idf/lines/line:RTP:1197611/routes
+
+* want a tiny response, just add "depth=0"
```
Comment (pbougue) [inline documentation/navitia.io/source/integration.rst]: Case consistency (last one also) ?

File: documentation/navitia.io/source/integration.rst [hunk 2]
```diff
@@ -362,6 +362,41 @@ Specific parameters
...
+
+	* https://api.navitia.io/v1/coverage/fr-idf/lines/line:RTP:1197611/routes?depth=2
+	* the response is a little more verbose (with some geojson appear in response)
+
+* Wanna fat more informations, let's try "depth=3"
+
+	* https://api.navitia.io/v1/coverage/fr-idf/lines/line:RTP:1197611/routes?depth=3
+	* big response: all stop_points are shown
+
+* wanna spam the internet bandwidth? Try "depth=42"
```
Comment (pbougue) [inline documentation/navitia.io/source/integration.rst]: Asking the question of usefulness of this (and just asking).
Is it stable? Do we want to communicate on it as definitively fixed?
But as it's fun to read, it makes the doc more readable :)

**Comment by pbougue** [comment]
:+1: for this doc, that have been missing for long time!

File: documentation/navitia.io/source/integration.rst (hunk 2 above)
Comment (antoine-de) [inline documentation/navitia.io/source/integration.rst]: I think it's ok, we can change it if needed, but the limit has to be in the doc.

And I quite like the way it is written.

**Pull request merged**